pkg/tfbridge/log.go

```diff
@@ -114,3 +118,49 @@ func (lr *LogRedirector) Write(p []byte) (n int, err error) {
 
 	return written, nil
 }
+
+// hostLogger is parameterized by L to allow `Status` to return the native Log type of a
+// stable Logger.
+type hostLogger struct {
+	ctx  context.Context
+	host *provider.HostClient
+	urn  resource.URN
+
+	status bool
+
+	err *multierror.Error
+}
+
+func (l *hostLogger) f(severity diag.Severity, msg string) {
+	f := l.host.Log
+	if l.status {
+		f = l.host.LogStatus
+	}
+	err := f(l.ctx, severity, l.urn, msg)
+	if err != nil {
+		l.err.Errors = append(l.err.Errors, err)
```

Comment:
Looks like we're accumulating errors from being unable to send a log message to the engine, perhaps because the engine is shutting down. Why? Would we ever want to change control flow behavior based on not being able to reach the engine? An alternative here is to dump it to glog low-level stream at verbose 9 level and proceed as if nothing happened.